### ISSUE TITLE
Issue on page /supervised/linear.html

### ISSUE THREAD

**Issue by MatildeRFonseca** [statisticalbiotechnology/dsbook#3]
Found a small typo in this part - marked the word with bold: 

"2.3. Notes on Optimization
The scipy.optimize.minimize uses a variant of Gradient descent to find an optimal set of parameters (a,b). Gradient **Decent** is an iterative optimization algorithm used in finding the minimum of a function. Here, we use it for finding the parameters that minimize the loss function."

Hope it helps - sorry if there was a better format to point it out, couldn't figure out how to do a pull request!

**Comment by percolator** [comment]
Oh, that is a decent comment. It is excellent that you found and used the issue tracker. Well appreciated!

Closed by cc38b0b0bdee19584aac3ad47507c1ea1e520188
